sympy/physics/mechanics/tests/test_particle.py

```diff
@@ -1,15 +1,30 @@
-from sympy.core.symbol import symbols
+from sympy.core.backend import symbols, Symbol
+from sympy.physics.vector import Vector
 from sympy.physics.mechanics import Point, Particle, ReferenceFrame, inertia
-
+from sympy.physics.mechanics.body_base import BodyBase
 from sympy.testing.pytest import raises, warns_deprecated_sympy
 
 
+def test_particle_default():
+    # Test default
+    p = Particle('P')
+    assert p.name == 'P'
+    assert p.mass == Symbol('P_mass')
+    assert p.masscenter.name == 'P_masscenter'
+    assert p.potential_energy == 0
+    assert p.__str__() == 'P'
+    assert p.__repr__() == ("Particle('P', masscenter=P_masscenter, frame=None,"
```

Comment:
Is this correct/are we happy with this? The repr should ideally return the code that, when pasted into the console, reinstantiates the same object. `P_masscenter` is just the `Particle`'s name as not the repr for `p`.

Reply:
`P_masscenter` would be something that should then be fixed in the `Point` class. The difficulty here would also lie in the fact that a `Point` has an internal dictionaries (as it is a node in multiple graphs).